src/routes/column.js

```diff
@@ -88,12 +88,16 @@ router.route('/')
   .get(async (req, res) => {
     try {
       let board;
+      let editable = true;
       if (req.query.id) {
         board = await Board.findById(req.query.id).populate({ path: 'columns', populate: { path: 'tasks' } });
+        if (board.authorId !== req.parsedToken.mongoId) {
+          editable = false;
+        }
       } else {
         board = await Board.findOne({ authorId: req.parsedToken.mongoId }).populate({ path: 'columns', populate: { path: 'tasks' } });
       }
-      res.send({ columns: board.columns, boardId: board._id });
+      res.send({ columns: board.columns, boardId: board._id, editable });
```

Comment:
Isn't it easier just to send `editable: board.authorId === req.parsedToken.mongoId` instead of creating additional local variable?

Reply:
There is no need to perform that comparison after searching for board by { authorId: req.parsedToken.mongoId } but okay.